### ISSUE TITLE
scrubbing iOS 12 

### ISSUE THREAD

**Issue by daslicht** [sampotts/plyr#1209]
Hello,
please try tge following on an iPad with iOS 12:

https://bit.ly/2A6FkzJ

Anything works fine except when you try to wind forward. 
I tryed adding RangeTool but that doesn't help. 

It is nearby impossible to scrub through the video using the track slider.

Idea ?

**Comment by jamesoflol** [comment]
To provide another data point, I've got this issue too. I have been meaning to look into it, but struggling with time ATM.

**Comment by jamesoflol** [comment]
@sampotts if you could please confirm you don't already have this one in-progress or figured out, I can do my best to make some time?

**Comment by sampotts** [comment]
Hey @jamesoflol I've not looked yet. It seems ok on my iPhone XS on 12.0.1. Would need to give it a go on an iPad.

**Comment by jamesoflol** [comment]
My bad, it looks like adding rangetouch does fix this for me.

Though the stock plyr range slider seems to be significantly more broken than a stock standard range slider in iOS. Normal sliders are a bit fiddly, but the plyr one is completely unusable for me without rangetouch.

Would be nice to fix, but not a priority for me. I'm happy with rangetouch - great little util!

**Comment by sampotts** [comment]
Yeah, the issue seems to lie with `<input type="range">` - I don't know why but they seemed to have not implemented support for it properly at all. A real pain.

**Comment by daslicht** [comment]
I have RangeTouch in my example  but it still dont work here on my ipad

**Comment by jamesoflol** [comment]
OK, right you are. Sorry, I only tested on my own. Same problem for me on iphone ios12 with your example.

**Comment by levi0214** [comment]
I have the same problem. 

[plyr.io](plyr.io) is fine, but [the demo on Codepen](http://codepen.io/sampotts/pen/jARJYp) is not.
@sampotts you can try to open the Codepen demo in iPhone.
